spec/Section 7 -- Response.md

```diff
@@ -107,14 +107,8 @@ syntax element.
 If an error can be associated to a particular field in the GraphQL result, it
 must contain an entry with the key `path` that details the path of the response
 field which experienced the error. This allows clients to identify whether a
-`null` result is intentional or caused by a runtime error.
-
-If present, this field must be a list of path segments starting at the root of
-the response and ending with the field associated with the error. Path segments
-that represent fields must be strings, and path segments that represent list
-indices must be 0-indexed integers. If the error happens in an aliased field,
-the path to the error must use the aliased name, since it represents a path in
-the response, not in the request.
+`null` result is intentional or caused by a runtime error. The value of this
+field is described in the [Path](#sec-Path) section.
```

Comment:
We seem to be using "entry" and "field" interchangeably in the old text, even though we also refer to the GraphQL "field". Let's take this opportunity to tighten it up by sticking to "entry" consistently when talking about the property on an error, and "field" when talking about a GraphQL object field.

```suggestion
entry is described in the [Path](#sec-Path) section.
```